etc/config/compiler-explorer.amazon.properties

```diff
@@ -36,7 +36,7 @@ sentrySlowRequestMs=30000
 alwaysResetLdPath=true
 plogConverter=/opt/compiler-explorer/pvs-studio-latest/bin/plog-converter
 
-cmake=/opt/compiler-explorer/cmake/bin/cmake
```

Comment:
this can remain the same, this is a link to the "latest" / version that we use for IDE mode cmake